javascript-arrays/main.js

```diff
@@ -0,0 +1,24 @@
+var colors;
+colors = ['red', 'white', 'blue'];
```

Comment:
You can do this all on one line, there's no need to separate this to multiple lines.